### ISSUE TITLE
EnderIO conduits will not extract all products from centrifuge into item filter

### ISSUE THREAD

**Issue by SachariasLofqvist** [GTNewHorizons/GT-New-Horizons-Modpack#17598]
### Your GTNH Discord Username

ArmedMumin

### Your Pack Version

2.6.1

### Your Server

Private Server

### Java Version

Java 21

### Type of Server

Don't know

### Your Expectation

Centrifuging ashes and with conduits exporting products from the centrifuge into item filters for later sorting into a storage system.

### The Reality

Centrifuged the ashes and sometimes the products stay in the centrifuge instead of being extracted and inserted into the item filters, however sometimes it works perfectly without any mishaps.
![2024-10-09_19 47 56](https://github.com/user-attachments/assets/764e5481-c8e4-4e8b-bbd1-fb0da930fbcc)
![2024-10-09_19 48 01](https://github.com/user-attachments/assets/0c4396bd-dc92-4df0-b138-4eb4139f3b25)
![2024-10-09_19 47 38](https://github.com/user-attachments/assets/b06e9162-6b90-429a-974a-7c890b2aa91b)

### Your Proposal

Fix the bug so the item conduit can extract all the items from the centrifuge and insert them into item filters.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by chochem** [comment]
![image](https://github.com/user-attachments/assets/c8c56ca5-1e6b-48d3-ae7f-1db71e50c922)
for real or did you just click because they are there?

**Comment by SachariasLofqvist** [comment]
Yes for real haha, if i try to output from my centrifuge directly to the item filter it sometimes but more often than not does not extract all the items from it and yes I asked in the discord before I posted this and I talked to a person who had experience the same kind of anomaly as me. However I have solved my issue with a chest before the itemfilter so it does not input directly into the itemfilter from the centrifuge, but it is annoying that outputting directly does not work.

**Comment by SachariasLofqvist** [comment]
Yes for real haha, if i try to output from my centrifuge directly to the item filter it sometimes but more often than not does not extract all the items from it and yes I asked in the discord before I posted this and I talked to a person who had experience the same kind of anomaly as me. However I have solved my issue with a chest before the itemfilter so it does not input directly into the itemfilter from the centrifuge, but it is annoying that outputting directly does not work.
